edx_prefectutils/s3.py

```diff
@@ -8,6 +8,26 @@
 from prefect.utilities.aws import get_boto_client
 
 
+@task
+def delete_s3_directory(bucket: str = None, prefix: str = None, credentials: dict = None):
```

Comment:
nit on the naming of this: does it actually delete the directory, or will it leave an empty prefix?

Reply:
If you delete all the keys under a prefix, the prefix is removed as well by S3.

Reply:
thanks.